packages/fxa-settings/src/pages/SigninTokenCode/index.tsx

```diff
@@ -0,0 +1,163 @@
+      //   'signin-token-code-error-general',
+      //   'Invalid confirmation code'
+      // );
+      // alertBar.error(errorSigninTokenCode);
+    }
+  };
+
+  return (
+    // TODO: redirect to force_auth or signin if user has not initiated sign in
+
+    // TODO: handle bounced email
+    //       if the account no longer exists, redirect to sign up
+    //       if the account exists, notify that the account has been blocked
+    //       and provide correct support link
+    <>
+      <header>
+        <FtlMsg id="signin-token-code-heading">
+          <h1 className="card-header">
+            Enter confirmation code
+            <span className="card-subheader"> for your Firefox account</span>
+          </h1>
+        </FtlMsg>
+      </header>
+
+      <main>
+        <div className="flex justify-center mx-auto">
+          <MailImg className="w-3/5" role="img" />
+        </div>
+
+        <FtlMsg id="signin-token-code-instruction">
+          <p id="verification-email-message" className="m-5 text-sm">
+            Enter the code that was sent to {email} within 5 minutes.
+          </p>
+        </FtlMsg>
+
+        <form
+          noValidate
+          className="flex flex-col gap-4"
+          onSubmit={handleSubmit(onSubmit)}
+        >
+          {/* Using `type="text" inputmode="numeric"` shows the numeric pad on mobile and strips out whitespace on desktop. */}
+          <FtlMsg id="signin-token-code-input">
+            <InputText
+              type="text"
+              inputMode="numeric"
+              label="Confirmation code"
+              placeholder="Enter 6-digit code"
+              onChange={(e) => {
+                setConfirmationCode(e.target.value);
+                // clear error tooltip if user types in the field
+                if (tokenErrorMessage) {
+                  setTokenErrorMessage('');
+                }
+              }}
+              onFocusCb={onFocusMetricsEvent ? onFocus : undefined}
+              errorText={tokenErrorMessage}
+              autoFocus
+              // TODO: validate pattern
+              pattern="\d[ ]*"
+              className="text-start"
+              anchorStart
+              autoComplete="off"
+              spellCheck={false}
+              prefixDataTestId="signin-token-code"
+              required
+              tooltipPosition="bottom"
+            />
+          </FtlMsg>
+
+          <FtlMsg id="signin-token-code-confirm-button">
+            <button
+              type="submit"
+              id="use-logged-in"
+              className="cta-primary cta-xl"
+            >
+              Confirm
+            </button>
+          </FtlMsg>
+        </form>
+        <div className="animate-delayed-fade-in opacity-0 mt-5 text-grey-500 text-xs inline-flex gap-1">
+          <FtlMsg id="signin-token-code-code-expired">
+            <p>Code expired?</p>
+          </FtlMsg>
+          {/* TODO: add button handling for resend code action
```

Comment:
What do we mean by button handling here? Like the action/cb? What are your thoughts on having this be a button with link styling?

Reply:
Yes, that makes sense! I dug a bit more into the resend_code mixin, and it looks like it displays the result in an alert bar and doesn't redirect. I'll update this to a button with link styling, as suggested!